cobra/preprocessing/categorical_data_processor.py

```diff
@@ -65,14 +65,14 @@ class CategoricalDataProcessor(BaseEstimator):
                   "scale_contingency_table", "forced_categories"]
 
     def __init__(self,
-                 model_type: str = "classification",
-                 regroup: bool = True,
-                 regroup_name: str = "Other",
-                 keep_missing: bool = True,
-                 category_size_threshold: int = 5,
-                 p_value_threshold: float = 0.001,
-                 scale_contingency_table: bool = True,
-                 forced_categories: dict = {}):
```

Comment:
I'm also used to this style, is that bad towards linters or something?

Reply:
To be honest, I don't know what the recommended style is. I started doing it at some point, then saw that PyCharm prefers it with spaces, but kept on going. At some point we'll enforce consistent code formatting as put forward in #54.

Reply:
I'd use what PyCharm suggests, normally it checks for PEP rules, and linters as well. We'll see when we tackle #54.